### DISCUSSION TITLE
How to generate CSS out of HTML string?

### DISCUSSION THREAD

**Discussion by pySilver** [tailwindlabs/tailwindcss#10405] [Help]
Is there an easy easy way to generate CSS from `raw HTML string`? My use case is having Page Builder where user may use any Tailwind classes. What I'd like to do is to call some JS command from my backend code, providing user generated HTML so I can receive CSS output in response, so I can save it to db and use as inline css.

**Comment by wongjn** [comment]
You can pass [raw content](https://tailwindcss.com/docs/content-configuration#configuring-raw-content) to be parsed.

**Reply by wongjn** [reply]
> @wongjn is it possible to pass config as string to build command then? I doubt that :/[^1]

Maybe, depending on your build system.

[^1]:https://github.com/tailwindlabs/tailwindcss/discussions/10405#discussioncomment-4767646

**Comment by pySilver** [comment]
@wongjn is it possible to pass config as string to build command then? I doubt that :/

**Reply by wongjn** [reply]
Replied at https://github.com/tailwindlabs/tailwindcss/discussions/10405#discussioncomment-4767693 to keep the reply chain together.
